libs/core/src/lib/toolbar/toolbar.component.ts

```diff
@@ -118,6 +125,10 @@ export class ToolbarComponent implements OnInit, AfterViewInit, OnDestroy, After
     @ContentChildren(forwardRef(() => ToolbarItemDirective))
     toolbarItems: QueryList<ToolbarItemDirective>;
 
+    /** @hidden */
+    @ContentChild(TitleComponent)
+    titleComponent: TitleComponent;
```

Comment:
Querying like this prevent from tree-shaking `TitleComponent` when compiling this component.

Here some more about it https://angular.io/guide/lightweight-injection-tokens

Reply:
done